Release 2022.06.2 (#3431)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

## June

### Weekly release 2022.06.2

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|
| [Bugs fixed in 2022.06.2](#bugs-fixed-in-2022-06-2)                      | June 23, 2022            |

#### Bugs fixed in 2022.06.2

* When an OAuth 2.0 client was created with a missing JWKS RSA modulus value (`n` parameter), the JWKS validation failed. (OKTA-424664)

> * Making a POST reset factors request to `/api/v1/users/${userId}/lifecycle/reset_factors` didn't reset the enrolled phone authenticator. (OKTA-463900)

Reviewer: ```suggestion
* A POST reset factors request to `/api/v1/users/${userId}/lifecycle/reset_factors` didn't reset the enrolled phone authenticator. (OKTA-463900)
```

* When the `maxSessionLifetimeMinutes` property of the Policy API Signon Session object was set, the value wasn't enforced. (OKTA-480442)

> * Using the JSON Web Key API or the Client Credentials API to manage client credentials didn't fire System Log events. (OKTA-494619)

Reviewer: ```suggestion
* When the JSON Web Key API or the Client Credentials API was used to manage client credentials, the event wasn't triggered in the System Log. (OKTA-494619)
```

> * When the Progressive Profiling feature was enabled, calling the `/introspect` endpoint changed the state handle incorrectly. (OKTA-502233)

Reviewer: ```suggestion
* When the Progressive Profiling feature was enabled, a request to the `/introspect` endpoint changed the state handle incorrectly. (OKTA-502233)
```
